src/apiCalls.js

```diff
@@ -18,11 +20,29 @@ function getSpecificCustomer(id) {
   .then(response => response.json())
 }
 
+function postBooking(booking) {
+  return fetch('http://localhost:3001/api/v1/bookings', {
+    method: "POST",
+    body: JSON.stringify(booking),
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  })
+  .then(response => {
+    if(!response.ok) {
+      throw new Error(response.status);
+    } else {
+      resolvePost();
+    }
+  })
+  .catch(error => console.log(error));
+}
+
 function getData() {
   const allCustomers = getAllCustomers()
   const allRooms = getAllRooms()
   const allBookings = getAllBookings()
   return Promise.all([allCustomers, allRooms, allBookings])
 }
```

Comment:
You could also refactor this as:

function getData() {
  return Promise.all([getAllCustomers(), getAllRooms(), getAllBookings()])
}